misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormatterUtils.java

```diff
@@ -79,4 +85,55 @@ static void sortImportDeclarations(List<ImportDeclarationNode> importDeclaration
                         node2.moduleName().stream().map(node -> node.toString().trim()).collect(Collectors.joining()))
                 .toComparison());
     }
+
+    /**
+     * Swap leading minutiae of the first import in the code and the first import when sorted.
+     *
+     * @param firstImport First ImportDeclarationNode in user code
+     * @param imports     Sorted formatted ImportDeclarationNode nodes
+     */
+    static void swapLeadingMinutiae(ImportDeclarationNode firstImport, List<ImportDeclarationNode> imports) {
+        String firstImportStr = getImportString(firstImport);
+        int prevFirstIndex = -1;
+        for (int i = 0; i < imports.size(); i++) {
+            if (firstImportStr.equals(getImportString(imports.get(i)))) {
+                prevFirstIndex = i;
+                break;
+            }
+
+        }
+        if (prevFirstIndex > 0) {
+            // remove comments from the previous first import
+            ImportDeclarationNode prevFirst = imports.get(prevFirstIndex);
```

Comment:
```suggestion
            ImportDeclarationNode prevFirstImportNode = imports.get(prevFirstIndex);
```
Shall we update variable names to be a little more self-explanatory?